src/platform/plugins/shared/workflows_management/public/shared/ui/next_execution_time.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import type { WorkflowTrigger } from '../../../server/lib/schedule_utils';
+import { getWorkflowNextExecutionTime } from '../../lib/next_execution_time';
+
+interface NextExecutionTimeProps {
+  triggers: WorkflowTrigger[];
+}
+
+export function NextExecutionTime({ triggers }: NextExecutionTimeProps) {
+  const nextExecutionTime = getWorkflowNextExecutionTime(triggers);
+
+  if (!nextExecutionTime) {
+    return null;
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate('workflows.workflowList.nextExecutionTime.tooltip', {
+        defaultMessage: 'Next execution: {date}',
+        values: {
+          date: nextExecutionTime.toLocaleString(),
```

Comment:
Kibana has settings for datetime format, I created a function to format the full date time according to user-set format, you can use it like that:

```tsx
const getFormattedDateTime = useGetFormattedDateTime();
//...
getFormattedDateTime(nextExecutionTime)
```

example:
https://github.com/elastic/kibana/blob/a685644d355913d4f96f73772cc8a92698d4b170/src/platform/plugins/shared/workflows_management/public/features/workflow_execution_detail/ui/workflow_step_execution_flyout.tsx#L176